caas/broker.go

```diff
@@ -104,8 +104,8 @@ type Broker interface {
 	EnsureOperator(appName, agentPath string, config *OperatorConfig) error
 
 	// OperatorExists returns true if the operator for the specified
-	// application exists.
-	OperatorExists(appName string) (bool, error)
+	// application exists, and whether the operator is terminating.
+	OperatorExists(appName string) (bool, bool, error)
```

Comment:
would it be good to have a struct here because two same types might be missed used?

Reply:
ok